recipes/libhdfs3/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "2.2.31" %}
+
+package:
+  name: libhdfs3
+  version: "{{ version }}"
+
+source:
+    git_url: https://github.com/PivotalRD/libhdfs3.git
+    git_tag: v{{ version }}
+
+build:
+  number: 0
+  # This package is a dependency of hdfs3, which is primarily used with HDFS on Linux.
+  skip: True  # [not linux]
+
+requirements:
+    build:
+        - protobuf
+        - libgsasl
+        - libntlm
+        - libxml2
+        - krb5
+        - libuuid
+        - boost ==1.60
```

Comment:
We are doing `1.60.*` and no `==`. Now I don't have a recent Boost ABI report, but this [one]( http://upstream.rosalinux.ru/versions/boost.html ) is pretty close. Basically every major minor pair is at risk or breaks it. While it is very rare they change the patch release, it does seem the few shown preserve ABI.